Directory.Build.props

```diff
@@ -19,7 +19,6 @@
     <Company>Microsoft Corporation</Company>
     <Copyright>Copyright © Microsoft Corporation. All rights reserved.</Copyright>
     <Deterministic>true</Deterministic>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Can we create a bug to fix the docs generation if this doesn't work?

Reply:
https://github.com/microsoft/fhir-server/issues/1681